src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -665,6 +665,7 @@ export class SavedObjectsRepository {
     }
 
     const deleteDocNotFound = body.result === 'not_found';
+    // @ts-expect-error @elastic/elasticsearch doesn't declare error on DeleteResponse
```

Comment:
@delvedor What PR caused this change? Doesn't the client return `body.error` if the `ignore` flag is specified?